utils/python/CIME/XML/compilerblock.py

```diff
@@ -0,0 +1,232 @@
+"""
```

Comment:
This really should be a part of the compilers class to best fit with the rest of cime